SandWorm/Analytics/Slope.cs

```diff
@@ -9,12 +9,32 @@ namespace SandWorm.Analytics
 {
     public class Slope : Analysis.MeshColorAnalysis
     {
-        private readonly int slopeCap = 500; // As measuring in % need an uppper limit on the value
+        readonly int maximumSlope = 1000; // Needs to be high to account for stray values
```

Comment:
Slope analysis is easy to break. No smoothing and zero frames averaging trips it almost immediately. When smoothing and averaging are on, bringing your hands very close to the Kinect sensor results in the following error message. 
`1. Solution exception:Index was outside the bounds of the array.`
Not sure whether this is part of the lookupTable logic or the CalculateSlope function.

Reply:
The issue (which is also present with the elevation measuring) is that the lookup table's indices have an implicit range of 'normal' values which are the bounds being exceeded. As slope is a % / asymptotic value exceeding that range can be triggered by any non-flat-ish inputs. 

A previous version of `GetColorCloudForAnalysis` has some bounds checking. I removed that with intent to try and benchmark whether its faster to use bounds checking, a try/catch, a modification to `CalculateSlope` or some other option. I'll go give that a shot now.